content/tracing/services/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Services list
+kind: Documentation
+further_reading:
+- link: "/tracing/services/service"
+  text: Learn more about services in Datadog
+- link: "/tracing/services/resource"
+  text: "Dive into your resource performances and traces"
+- link: "/tracing/services/trace"
+  text: "Understand how to read a Datadog Trace"
+---
+
+## Overview
+
+After you have [instrumented your application](/tracing/setup), your reporting services appear on [the APM services page](https://app.datadoghq.com/apm/services).  
+Select a service view detailed performance insights, [read our dedicated service documentation to learn more](tracing/services/service)
+
+{{< img src="tracing/services/services_page.png" alt="Services page" responsive="true" popup="true">}}
+
+## Filtering the service list
+
+The services list is a bird's eyed view of all [services](/tracing/services/service) reporting from your infrastructure. It can be filtered depending on:
+
+* [Your environment](/tracing/setup/environment)
+* [Your service type](#services-types)
+* A query (basic text filtering).
+
+{{< img src="tracing/services/services_filtering.gif" alt="Services filtering" responsive="true" popup="true" style="width:75%;">}}
```

Comment:
I don't think this image is adding anything.  It also has a lot of blank space and the cursor hanging around.

Reply:
it's a gif, right now gif are not running on staging env, but it should work on prod